bike_rent/views/res_partner_views.xml

```diff
@@ -30,12 +30,12 @@
     </field>
 </record>
 
-<record id="bike_rent.users_action" model="ir.actions.act_window">
-    <field name="name">Users</field>
+<record id="bike_rent.partner_action" model="ir.actions.act_window">
```

Comment:
module declaration is unnecessary, it is saved by default to `ir.model.data`, drop the `bike_rent.` prefix